src/screens/Assets/WalletView.js

```diff
@@ -494,12 +503,6 @@ class WalletView extends React.Component<Props, State> {
             />)}
           {!isInSearchMode && (!balance || !!showFinishSmartWalletActivation) &&
           <ActionsWrapper>
-            {!!showFinishSmartWalletActivation && smartWalletFeatureEnabled &&
-            <ListItemChevron
-              label={smartWalletDeployLabel}
-              onPress={() => navigation.navigate(SMART_WALLET_INTRO, { deploy: showDeploySmartWallet })}
```

Comment:
That was covering the case for legacy users when they created the sw account, but haven't deployed it yet

Reply:
Do you mean the case like this:
- on the onboarding page, select "create account"
- you'll have the sw account created but not deployed yet
- you'll just see the insights "Why Smart Wallet knocks out your old private key wallet?" and then "Activate Smart Wallet" and you can deploy wallet from there which I think is correct

Reply:
You need to a) transfer eth to cover the deployment; b) deploy the account.
Since it's a 2 step operation - it could fail in between, that's why we've added the button to retry the deployment.
I think it could be possible if the user transfers too little eth, just to cover the gas and then the gas price spikes